tests/ignite/metrics/test_cohen_kappa.py

```diff
@@ -50,8 +51,9 @@ def test_input_types():
         ck.update((torch.randint(0, 2, size=(10,)).long(), torch.randint(0, 2, size=(10, 5)).long()))
 
 
-def test_check_shape():
-    ck = CohenKappa()
+def test_check_shape(available_device):
```

Comment:
Do not add available_device to the test that checks assertions with `with pytest.raises` (mentioned in the issue description)